.github/workflows/buildMain.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
```

Comment:
Why are these two workflow files part of this PR? Maybe you have to update your fork and create a new PR.

Reply:
No clue about when these files appeared in my PR. I will update my fork. Shall I delete this PR and create a new one?

Reply:
Yes, I think this should solve the problem